services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -1458,26 +1459,22 @@ func TestTransformer(t *testing.T) {
 		},
 		{
 			Name:         "Lock env AND app and then Deploy and unlock both locks",
-			Transformers: makeTransformersDoubleLock(api.LockBehavior_Queue, true),
+			Transformers: makeTransformersDoubleLock(api.LockBehavior_Record, true),
 			Test: func(t *testing.T, s *State) {
 				// check that the state reads the correct versions
 				i, err := s.GetEnvironmentApplicationVersion("production", "test")
 				if err != nil {
 					t.Fatal(err)
 				}
-				if i == nil {
-					t.Errorf("unexpected version: expected 1, actual nil")
-				} else {
-					if *i != 1 {
-						t.Errorf("unexpected version: expected 1, actual %d", *i)
-					}
+				if i != nil {
+					t.Errorf("unexpected version: unexpected nil")
 				}
 				q, err := s.GetQueuedVersion("production", "test")
 				if err != nil {
 					t.Fatal(err)
 				}
-				if q != nil {
-					t.Errorf("unexpected version: expected nil, actual %d", *q)
+				if q == nil {
+					t.Errorf("unexpected version: expected %d, actual nil", *q)
```

Comment:
I think the expected value here is 1. 
```suggestion
					t.Errorf("unexpected version: expected 1, actual nil")
```